### PR TITLE
GP2-1618: Migrate transition period contact pages

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset ports the TP/EU Exit contact page and its success page to the new CMS. It retains support for making some content on both pages manageable via the CMS (via a new Non-Page Content Snippet), but (with permission from the PO) ditches support for CMS-driven control over the form labels, which makes things simpler than before.

As with much of this migration, "simpler" != "simple", so this work also involved chasing down whether the end result of various mixins, etc were needed. The things that have been ditched are noted in code comments and/or git logs

This changeset also contains a data migration to bootstrap the content needed for the form page and the success page.

I also took the opportunity to start moving some view and form tests in domestic into subdirs

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1618
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Explains how to test locally, including how to set up appropriate data

* pull branch locally with all supporting services running - including directory-forms-api, with config to talk to dev/staging Zendesk
* go to http://greatcms.trade.great:8020/transition-period/contact/and complete the workflow

- [X] Includes screenshot(s) 

Screenshot 2021-06-08 at 17 01 53

Screenshot 2021-06-08 at 17 02 19

Screenshot 2021-06-08 at 17-03-45 Transition period enquiries - great gov uk

Screenshot 2021-06-08 at 17-03-56 Thank you for submitting your question - great gov uk

### Housekeeping

- [X] Added all new environment variables to Vault - dev, stage, UAT and prod
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [X] Frontend assets have been re-compiled - by @lewis-coulson-dit 

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Review by bobmeredith — APPROVED** [review]
LGTM, although TBH, it's way over my attention span :)

File: domestic/views/euexit.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from django.conf import settings
+from django.urls import reverse_lazy
+from django.views.generic import TemplateView
+from django.views.generic.edit import FormView
+
+from core import snippet_slugs
+from core.mixins import GetSnippetContentMixin, PrepopulateFormMixin
+from domestic import forms
+
+SESSION_KEY_FORM_INGRESS_URL = 'FORM_INGRESS_URL'
```
Comment (webbyfox) [inline domestic/views/euexit.py:11]: Is this URL meant to come from any variable (i.e. env) - Hard coding string?

**Review by webbyfox — APPROVED** [review]
minor comment but otherwise looks good

File: domestic/views/euexit.py (hunk 1 above)
Comment (stevejalim) [reply domestic/views/euexit.py:11]: It's a key set in the session by BaseInternationalContactFormView and the data comes from the HTTPRequest

```
    def get(self, *args, **kwargs):
        self.request.session[SESSION_KEY_FORM_INGRESS_URL] = self.request.META.get('HTTP_REFERER')
        return super().get(*args, **kwargs)
```

**Review by stevejalim — COMMENTED** [review]
self-review - seems fine

**Pull request merged**